crd/crd.go

```diff
@@ -81,21 +91,15 @@ func NewDatabaseCRD() *apiextv1.CustomResourceDefinition {
 											Minimum:     floatptr(20),
 											Maximum:     floatptr(64000),
 										},
-										"MaxAllocatedSize": {
-											Type:        "integer",
-											Description: "Database size in Gb",
-											Minimum:     floatptr(20),
-											Maximum:     floatptr(64000),
-										},
 										"multiaz": {
 											Type:        "boolean",
 											Description: "should it be available in multiple regions?",
 										},
-										"publiclyaccessible": {
+										"publicaccess": {
```

Comment:
this must be the field name in the resource representation, not the field name on the Go struct.